docs/src/components/pricing/EarlyBird.tsx

```diff
@@ -6,10 +6,19 @@ import Stack from '@mui/material/Stack';
 import Button from '@mui/material/Button';
 import Link from 'docs/src/modules/components/Link';
 import KeyboardArrowRightRounded from '@mui/icons-material/KeyboardArrowRightRounded';
+import styled from '@emotion/styled';
```

Comment:
See the other pages:

```suggestion
import { styled } from '@mui/material/styles';
```